Makefile

```diff
@@ -1,7 +1,7 @@
 export GO111MODULE := on
 
 .PHONY: test
-test: generate vet
+test: vet
```

Comment:
This now no longer generates files before running tests... so you're potentially testing the old generated code.

Reply:
It caused me loss of work as during development I'm working on the generated code directly and then replicate the changes to the templates, because it's faster and there is no Go syntax highlighting in mustache template files. And having `make test` re-generating the files came unexpected to me, I don't think it's a common practice for `make test` to update the tested code itself.